looprpc/client.proto

```diff
@@ -1645,28 +1661,53 @@ enum DepositState {
     */
     WITHDRAWN = 3;
 
+    /*
+    LOOPING_IN indicates that the deposit is currently being used in a static
+    address loop-in swap.
+    */
+    LOOPING_IN = 4;
+
+    /*
+    LOOPED_IN indicates that the deposit was used in a static address loop-in
+    swap.
+    */
+    LOOPED_IN = 5;
+
+    /*
+    SWEEP_HTLC_TIMEOUT indicates that the deposit is part of an active loop-in
+    of which the respective htlc was published by the server and the timeout
+    path has opened up for the client to sweep.
+    */
+    SWEEP_HTLC_TIMEOUT = 6;
+
+    /*
+    HTLC_TIMEOUT_SWEPT indicates that the timeout path of the htlc has been
+    swept by the client.
+    */
+    HTLC_TIMEOUT_SWEPT = 7;
+
     /*
     PUBLISH_EXPIRED indicates that the deposit has expired and the sweep
     transaction has been published.
     */
-    PUBLISH_EXPIRED = 4;
+    PUBLISH_EXPIRED = 8;
 
     /*
     WAIT_FOR_EXPIRY_SWEEP indicates that the deposit has expired and the sweep
     transaction has not yet been sufficiently confirmed.
     */
-    WAIT_FOR_EXPIRY_SWEEP = 5;
+    WAIT_FOR_EXPIRY_SWEEP = 9;
```

Comment:
good q, PUBLISH_EXPIRED creates and publishes the expiry tx, WAIT_FOR_EXPIRY_SWEEP subscribes to the confirmation. I separated these in case of recovery, we don't have to construct and publish the tx again but just subscribe to it. Also if the construction fails f.w.r. we won't subscribe cause the state machine lands in an error.

Republishing of sweep txns will be added shortly.